### DISCUSSION TITLE
Main Chat

### DISCUSSION THREAD

**Discussion by L7NEG** [UltimateMenu/UltimateMenu#28] [General]
Welcome To Ultimate Menu Discussion

**Comment by SSGamerTag** [comment]


**Reply by L7NEG** [reply]
> Hello, please **what button I press to open Yimmenu** in GTA Online?

It is the insert key on your keyboard

**Reply by SSGamerTag** [reply]
It says I should press "VK_LButton", Microsoft says it is the left button of the mouse but it does not work, is there any way to change the settings outside the game in folders?

**Reply by L7NEG** [reply]
The menu Toggle key is insert key that is on the numpad side of your keyboard if it is 100%

**Comment by StakensJ** [comment]
What should I set as everyones cut in a heist, and does setting above 100% even work?

**Reply by L7NEG** [reply]
Setting above 100% on Cayo will pass the limit which is just 2.5m max money so just make it 100%

**Comment by sillyguy69** [comment]
Is anyone else getting an error when opening the menu with ultimate menu?
![image_2024-08-15_101900159](https://github.com/user-attachments/assets/c7e0fce4-1f27-4e54-bf51-63cb5ccaea18)

**Reply by L7NEG** [reply]
ok thx for letting me know i will do a quick fix for it

**Reply by L7NEG** [reply]
> Is anyone else getting an error when opening the menu with ultimate menu? ![image_2024-08-15_101900159](https://private-user-images.githubusercontent.com/178489675/358250610-c7e0fce4-1f27-4e54-bf51-63cb5ccaea18.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM3MzU5MjEsIm5iZiI6MTcyMzczNTYyMSwicGF0aCI6Ii8xNzg0ODk2NzUvMzU4MjUwNjEwLWM3ZTBmY2U0LTFmMjctNGU1NC1iZjUxLTYzY2I1Y2NhZWExOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxNVQxNTI3MDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kYjY0YzBhNmFhNDFjOGZhZTJkOTc0M2NkMzBjNTdlNmM2MjhiNmZkOWQzMzdkNDc4MjQ3YmU3MDQzOWE1NmQzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.-lvl_rMuRr4ypf75Ex-rfXqknQUwcahfcxsdNNpGl2Q)

bro this error is kiddions modest menu related just load into story or online and then click on reload scripts

**Comment by king69666** [comment]
Will there ever be an option to modify the bag limits in Cayo? I know that this can be done with a HE but I don't want to run Yim and an HE too.

**Reply by L7NEG** [reply]
it is in already fr in cayo extra menu in ultimate menu for yimmenu fr

**Reply by king69666** [reply]
Ok, yah I found it. I'm just blind it seems....

**Comment by ariavylre** [comment]
hello! i was wondering how to increase my crew level rank using this menu?

**Reply by L7NEG** [reply]
It is simple from recovery > set Crew Rank choose the rank you want then just simply click all the buttons to apply it in all 4 clan slots

**Reply by ariavylre** [reply]
ooh nice, thank you so much!

**Reply by L7NEG** [reply]
> ooh nice, thank you so much!

you are welcome bro <3

**Comment by JUFEGAAM** [comment]
Hello im getting this error and i dont know what to do, i've tried all i know, AV is disabled, i bypassed modest menu on the firewall, i ran as admin pls help
![image](https://github.com/user-attachments/assets/570b6b69-ebce-45b0-b562-012434e53ae5)

**Reply by L7NEG** [reply]
Did you disabled BE from rockstar launcher or not if not then try it

**Reply by JUFEGAAM** [reply]
i did not, because i wanna use them on gta online and you need BE for that

**Reply by JUFEGAAM** [reply]
> n

what could i do?

**Comment by yqq666zpx** [comment]
这个有Chinese版本吗

**Reply by yqq666zpx** [reply]
汉化版本

**Comment by spraY1965** [comment]
How can i put money and Rp to my account but not in local save

**Reply by L7NEG** [reply]
You have to risk by going into invite only session with yimmenu then you can set your rp and unlocks and it will be saved online

**Comment by Novikov25** [comment]
when i press the insert key the menu dosent pop up

**Reply by L7NEG** [reply]
if you are on laptop just press fn key then insert

**Reply by L7NEG** [reply]
or click the ins on the numpad section

**Comment by Byzanthine** [comment]
How to use this with yimmenu 2?

**Reply by L7NEG** [reply]
We just have to wait for a lua scripting to be added in yimmenuv2

**Comment by ITZAJ22** [comment]
wait is this still working?

**Reply by ghost** [reply]
Yes yimmenu is back under yimmenuv2 here https://github.com/YimMenu/YimMenuV2

**Reply by L7NEG** [reply]
as @iSk1P said its working but it dosent have lua support yet

**Reply by ITZAJ22** [reply]
ah I gotta wait for FSL v2 to drop to be safe, dont wanna get my account banned

**Reply by L7NEG** [reply]
> ah I gotta wait for FSL v2 to drop to be safe, dont wanna get my account banned

fsl is out fr for gta enhanced here 

# FSL V1.6


# Changelogs:
v1.6 - Updated for b813. Fixed ATMs & Fixed singleplayer save corruption

- [Unknowncheats File Link](<https://www.unknowncheats.me/forum/downloads.php?do=file&id=48947>)
- [Mirrored Link](https://files.l7neg.uk.to/fsl/1.6/version.dll)

- [What Is FSL And More Details Can Be Found Here](<https://www.unknowncheats.me/forum/grand-theft-auto-v/616977-fsl-local-gtao-saves.html>)

Notes: 
The latest version does not support the Legacy version of the game.

Known Bugs:
* Attempting to claim the free GTA+ vehicle from the Vinewood app would kick you out of the session. Use the in-game browser instead.
